packages/admin/dashboard/src/routes/orders/order-create-fulfillment/components/order-create-fulfillment-form/order-create-fulfillment-item.tsx

```diff
@@ -35,36 +35,91 @@ export function OrderCreateFulfillmentItem({
     item.product_id,
     item.variant_id,
     {
-      fields: "*inventory,*inventory.location_levels",
+      fields: "*inventory,*inventory.location_levels,*inventory_items",
     },
     {
       enabled: !!item.variant,
     }
   )
 
   const { availableQuantity, inStockQuantity } = useMemo(() => {
-    if (!variant || !locationId) {
+    if (
+      !variant?.inventory_items?.length ||
+      !variant?.inventory?.length ||
```

Comment:
question: Is the second one not an alias of the first one?

Reply:
`inventory` is an array of associated `InventoryItem` entities and `inventory_item` contains an array of `ProductVariantInventoyItem` entities which conatin `required_quantity` etc.

The lengths of the arrays should be the same but the check is for redundancy.